### PR TITLE
fix: no manifest found

### PR DESCRIPTION
**Pull request by siccous**
Small fix for edge case If no manifest can be found.

### CODE REVIEW

File: pytest_odoo.py [hunk 1]
```diff
@@ -237,7 +237,9 @@ def _find_manifest_path(collection_path: Path) -> Path:
     while level < 5  and not (path.parent / "__manifest__.py").is_file():
         path = path.parent
         level += 1
-    return path.parent / "__manifest__.py"
+    if (path.parent / "__manifest__.py").is_file():
```
Comment (petrus-v) [inline pytest_odoo.py:240]: Hum I think if collection_path already contain the manifest it should return it self. reading your code I think this is not the case. I've fixed that in a closed (pr #70) with unit-test !  

I'll pick my commits and create a new PR with that fix independantly

Comment (petrus-v) [reply pytest_odoo.py:240]: hum... the case would be, if someone add test_ file at the same level as __manifest__ I suppose!

Comment (petrus-v) [reply pytest_odoo.py:240]: here you are https://github.com/camptocamp/pytest-odoo/pull/73

**Comment by yvaucher** [comment]
Closing in favor of #73

**Pull request closed without merging**